data/reusables/actions/working-directory-must-exist.md

```diff
@@ -0,0 +1,5 @@
+{% note %}
+
+**Note:** If the `working-directory` does not exist, the step will probably fail to run the `shell`.
```

Comment:
```suggestion
**Note:** The `working-directory` must exist for the `shell` to run.
```
Just editing this to use less words and not express doubt (the word "probably"). 

Also, do you think this information is necessary? To me it seems reasonable to expect users (people well-versed enough to be in this doc and writing workflows) to understand that they cannot navigate to a directory that doesn't exist.

Reply:
I specifically tripped on this in a github owned workflow and it wasn't remotely intuitive. It was incredibly frustrating. It related to spooky-action-at-a-distance -- the working-directory wasn't set in the local step, so it wasn't obvious what was going wrong.

The `probably` is me hedging my bets -- if someone can confirm that it's definitely guaranteed not to work, then removing it is definitely the right choice.

Reply:
I'll take that.

I'd also love for you to take the rest of this to the engineering side to improve the experience or to the docs team to add a troubleshooting doc -- both are beyond my energy at this time.